sktime/tests/_config.py

```diff
@@ -85,6 +85,7 @@
 from sktime.transformations.series.compose import OptionalPassthrough
 from sktime.transformations.series.outlier_detection import HampelFilter
 from sktime.transformations.series.boxcox import BoxCoxTransformer
+from sktime.clustering.base.base import BaseCluster
```

Comment:
Could we make `BaseCluster` available in the parent directory by importing it in the init file?  
```suggestion
from sktime.clustering.base import BaseCluster
```
Perhaps also rename to `BaseClusterer`?